### ISSUE TITLE
Urgent pypi problem

### ISSUE THREAD

**Issue by peterbe** [eventable/vobject#2]
As of today, it's now no longer possible to `pip install vobject` version 0.8.1c

```
(testenv2.7) :/tmp/testenv2.7$ pip install vobject==0.8.1c
Collecting vobject==0.8.1c
  Could not find a version that satisfies the requirement vobject==0.8.1c (from versions: 0.8.1c-dist, 0.2.3, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.4.4, 0.4.5, 0.4.6, 0.4.7, 0.4.8, 0.4.9, 0.5.0, 0.6.6)
No matching distribution found for vobject==0.8.1c
```

This has been working for many many months and suddenly it doesn't. Now, if you just do `pip install vobject` it works but it installs version 0.6.6. Which is ancient. 

And I can't install the new file with the word "dist" in it:
```
(testenv2.7) :/tmp/testenv2.7$ pip install vobject==0.8.1c-dist
Collecting vobject==0.8.1c-dist
  Downloading vobject-0.8.1c-dist.tar.gz (55kB)
    100% |████████████████████████████████| 57kB 944kB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/1x/2hf5hbs902q54g3bgby5bzt40000gn/T/pip-build-mmoKUh/vobject/setup.py", line 43, in <module>
        from ez_setup import use_setuptools
    ImportError: No module named ez_setup

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/1x/2hf5hbs902q54g3bgby5bzt40000gn/T/pip-build-mmoKUh/vobject
```

**Comment by tianon** [comment]
I can get `pip install vobject==0.8.1c-dist` to run successfully if I first `pip install ez_setup`, but that's a bit hacky, and doesn't solve the version ordering issue. :smile:

**Comment by peterbe** [comment]
What is this `ez_setup` thing? it looks old.

**Comment by tianon** [comment]
Nice, looks like this was reverted on PyPI (and the source is updated in #3 too :+1:).

**Comment by skarim** [comment]
Sorry for all the issues everyone! Looks like there were a few discrepancies between the code previously hosted on PyPi and the repo. Just removed the `ez_setup` import so that issue is gone.

I'm not quite sure what is causing 0.6.6 to be the default for PyPi (only 0.8.1c is enabled)
![image](https://cloud.githubusercontent.com/assets/1701557/12658421/f5663450-c5d6-11e5-9051-1e0269d5cddc.png)

Perhaps creating/uploading a new version (0.8.1d) will solve that issue?

**Comment by tianon** [comment]
I think I actually spoke a little too soon, the pypi website shows the right version, but pip fails:
```console
root@db172281e603:/# pip install vobject==0.8.1c
Collecting vobject==0.8.1c
  Could not find a version that satisfies the requirement vobject==0.8.1c (from versions: 0.2.3, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.4.4, 0.4.5, 0.4.6, 0.4.7, 0.4.8, 0.4.9, 0.5.0, 0.6.6)
No matching distribution found for vobject==0.8.1c
```

**Comment by tianon** [comment]
So maybe there's actually still something wrong with the `0.8.1c` upload and `pip` is noticing that and ignoring it?

**Comment by skarim** [comment]
Just uploaded a new version `0.8.2` to PyPi with the `ez_install` fix from #3 
`pip install vobject` now works correctly again :smile: 

Sorry again for all the trouble!

**Comment by peterbe** [comment]
Thank you.
